src/test/regress/output/multi_subquery.source

```diff
@@ -861,7 +861,7 @@ DEBUG:  predicate pruning for shardId 270011
 DEBUG:  predicate pruning for shardId 270012
 DEBUG:  predicate pruning for shardId 270011
 DEBUG:  predicate pruning for shardId 270012
-DEBUG:  building index "pg_toast_17247_index" on table "pg_toast_17247"
```

Comment:
Well, this test still seems to fail. Probably you forgot to update after adding the test with `exclusion_const_wrong_operator ` table?